SimPEG/PF/MagAnalytics.py

```diff
@@ -189,7 +189,7 @@ def IDTtoxyz(Inc, Dec, Btot):
     """
     Bx = Btot*np.cos(Inc/180.*np.pi)*np.sin(Dec/180.*np.pi)
     By = Btot*np.cos(Inc/180.*np.pi)*np.cos(Dec/180.*np.pi)
-    Bz = -Btot*np.sin(Inc/180.*np.pi)
+    Bz = Btot*np.sin(Inc/180.*np.pi)
```

Comment:
just a quick sanity check - the sign change is intentional??

Reply:
Correct, we were previously using UBC sign convention with Z positive down. Now everything should be xyz right handed